src/Form.elm

```diff
@@ -233,15 +288,23 @@ update validation msg (F model) =
                         |> Maybe.map Tree.asList
                         |> Maybe.withDefault []
 
+                filterChangedFields =
+                    Set.filter (not << String.startsWith listName)
```

Comment:
Let me know if you'd like some help with this one - I can spend some more time looking at it as well.